DogTales/dog/player.cpp

```diff
@@ -27,6 +23,12 @@ void Player::tick(bave::Seconds const dt) {
 
 void Player::draw(bave::Shader& shader) const { m_sprite.draw(shader); }
 
+float const Player::get_controller_state(std::string key) {
```

Comment:
Returning a `float const` is redundant, compiler/linter should be warning about this.

Reply:
should I just make it `float`?

Reply:
Yep